src/main/java/com/apaliakou/Consequent.java

```diff
@@ -0,0 +1,176 @@
+package main.java.com.apaliakou;
+
+import java.util.SortedMap;
+
+class Data { // создал класс для исходных данных
+    void showmenu () { // создал метод отображения исходного меню
+        System.out.println(); // добавил пробел
+        System.out.println("МЕНЮ ПРОГРАММЫ, ОТВЕЧАЮЩЕЙ НА ЗАПРОС ПОЛЬЗОВАТЕЛЯ: ");
+        System.out.println("1. algorithmld - тип алгоритма;");
+        System.out.println("2. loopType - тип циклов, которые нужно использовать;");
+        System.out.println("3. n - параметр, передаваемый в алгоритм.\n");
```

Comment:
New line character in windows and linux are different. So you can use System.lineSeparator()